package.json

```diff
@@ -46,6 +48,18 @@
     "@types/react-headroom": "^2.2.1",
     "@types/react-helmet": "^6.1.1",
     "@types/rebass": "^4.0.8",
+    "@typescript-eslint/eslint-plugin": "^4.28.0",
+    "@typescript-eslint/parser": "^4.28.0",
+    "eslint": "^7.29.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^3.4.0",
+    "lint-staged": "^11.0.0",
+    "prettier": "^2.3.1",
     "typescript": "^4.3.4"
+  },
+  "lint-staged": {
+    "*.{js,ts,tsx}": [
+      "eslint --fix"
+    ]
```

Comment:
ℹ️ probably not used

Reply:
It's the same configuration that BPMN Visualization. 
It's used on commit, if you don't run the command to format the code.

Reply:
I haven't seen the husky configuration, so truly legit. :+1